src/style.css

```diff
@@ -0,0 +1,92 @@
+body {
+  font-family: "Montserrat", sans-serif;
+  overflow: hidden;
+  margin: -21px 0 0 0;
```

Comment:
I don't think setting the margin here in this way is a good idea, I mean, why specifically `-21px` anyway? I recommend you to remove all built-in margins and paddings from all elements with the `*` selector, and only add them where it's really necessary. Also, I believe you have set overflow to hidden to achieve the circular div trick, right? It makes more sense to me to hide the overflow within `.bg-one` instead of the whole body.